worldedit-core/src/main/java/com/sk89q/worldedit/command/ApplyBrushCommands.java

```diff
@@ -111,11 +116,19 @@ public void forest(CommandParameters parameters,
         name = "item",
         desc = "Use an item"
     )
+    @CommandPermissions("worldedit.brush.item")
     public void item(CommandParameters parameters,
                      Player player, LocalSession localSession,
                      @Arg(desc = "The type of item to use")
-                         BaseItem item) throws WorldEditException {
-        setApplyBrush(parameters, player, localSession, new ItemUseFactory(item));
+                         BaseItem item,
+                     @Arg(desc = "The direction to face applying the item", def = "up")
+                     @Direction(includeDiagonals = true)
+                         BlockVector3 direction) throws WorldEditException {
+        player.print(TextComponent.of("Warning: ", TextColor.RED)
```

Comment:
Any reason this isn't all one message? All the text is going to be red regardless, right?